.github/workflows/docs.yml

```diff
@@ -5,9 +5,10 @@ jobs:
   docs:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - uses: ammaraskar/sphinx-action@master
+      - uses: actions/checkout@v2
+      - uses: sphinx-toolbox/sphinx-action@master
```

Comment:
I guess this might be necessary if the docs dependencies are to be installed from root and not from within docs. I am worried with how obscure the fork of the Action is but we can't do much about this.